crates/theme_importer/src/vscode/converter.rs

```diff
@@ -11,6 +11,8 @@ use crate::ThemeMetadata;
 
 use super::ZedSyntaxToken;
 
+const ZED_THEME_SCHEMA_URL: &'static str = "https://zed.dev/schema/themes/v0.1.0.json";
```

Comment:
We really shouldn’t be regenerating new versions of the same schema.

Each time we make changes to the schema that should be a new version.